docs/warnings/create-element-types.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Invalid type for createElement
+layout: single
+permalink: warnings/create-element-types.html
+---
+
+You probably came to this page because of this error:
+
+```
+Warning: React.createElement: type should not be null, undefined, boolean, or number.
+It should be a string (for DOM elements) or a ReactClass (for composite components).
+```
+
+This usually occurs when attempting to render an element that is of an invalid type. The following examples will trigger this error:
+
```

Comment:
Most people will come to this page from the link provided in the warning message. Assuming that is true, I think a more informative message could be show here without repeating the warning.

Something like:
You probably came here because your code is trying to create a ReactElement with an invalid type. This usually happens when you have an invalid import statement.